tests/page-objects/default/contacts/contacts.wdio.page.js

```diff
@@ -235,21 +236,23 @@ const editDistrict = async (districtName, editedName) => {
   await submitForm();
 };
 
-const createNewAction = async (formName) => {
+const createNewAction = async (formName, waitForDisplayedForm = true) => {
   await (await newActionContactButton()).waitForDisplayed();
   await (await newActionContactButton()).waitForClickable();
   await (await newActionContactButton()).click();
-  await openForm(formName);
+  await openForm(formName, waitForDisplayedForm);
 };
 
-const openForm = async (name) => {
+const openForm = async (name, waitForDisplayed = true) => {
   const parent = await newActionContactButton().parentElement();
   await browser.waitUntil(async () => await parent.getAttribute('aria-expanded') === 'true');
 
   for (const form of await forms()) {
     if (await form.getText() === name) {
       await form.click();
-      await (await formTitle()).waitForDisplayed();
+      if(waitForDisplayed) {
```

Comment:
I am sure there are better ways to manage this, but this is the only idea that I had.
Any suggestions?

Reply:
Some options come to mind:
1. Change the parameter name to `shouldDisplayTitle` or `shouldDisplayContent` 
2. Don't ever check the content here and do that outside this function
3. Create another function (`openFormWithWarning`) that verifies a modal opens and returns the modal's content. So, the spec is in charge of checking the modal's message. 

I think the parameter `waitForDisplayed` isn't describing what is happening.

Reply:
Thanks Jenny.
I created the new function `openFormWithWarning` 😄